### PR TITLE
Block Central banking editorial desk

### PR DESCRIPTION
**Pull request by lokendersinghft**
# Description
Blocked Central banking editorial desk

## What

1. Blocked central banking editorial desk
2. Use opa container for integration testing instead of mocking the opa client

## Why

JIRA --> https://financialtimes.atlassian.net/browse/UPPSF-5504

## Scope and particulars of this PR (Please tick all that apply)

- [ ] Tech hygiene (dependency updating & other tech debt)
- [x] Bug fix
- [ ] Feature
- [ ] Documentation
- [ ] Breaking change
- [ ] Minor change (e.g. fixing a typo, adding config)

___
This Pull Request follows the rules described in our [Pull Requests Guide](https://github.com/Financial-Times/upp-docs/tree/master/guides/pr-guide)

### CODE REVIEW

File: pkg/message/message_handler.go [hunk 1]
```diff
@@ -133,7 +131,7 @@ func (h *Handler) handleMessage(msg kafka.FTMessage) {
 	}
 
 	if res.Skip {
-		log.WithField("reasons", res.Reasons).Infof("Skipping non FT Pink content")
+		log.WithField("reasons", res.Reasons).Infof("Skipping content")
```
Comment (todor-videv1) [inline pkg/message/message_handler.go:134]: **praise**: Good catch.

File: pkg/message/message_handler_test.go [hunk 2]
```diff
@@ -162,10 +159,13 @@ func mockMessageHandler(esClient ESClient, mocks ...interface{}) (es.AccessConfi
 
 	var handler *Handler
 
-	opaAgent := mockOpaAgent{
-		returnResult: &policy.ContentPolicyResult{},
+	paths := map[string]string{
+		policy.FilterSVContent: "content_rw_elasticsearch/content_msg_evaluator",
 	}
 
+	opaClient := opa.NewOpenPolicyAgentClient(os.Getenv("OPA_URL"), paths, opa.WithLogger(uppLogger))
```
Comment (todor-videv1) [inline pkg/message/message_handler_test.go:166]: **question**: Shouldn't we check whether the env variable is actually accessible (has been set)? What happens if you forget to set it and run the tests?

File: pkg/message/message_handler_test.go [hunk 3]
```diff
@@ -276,6 +276,7 @@ func TestHandleWriteMessageFromBodyXML(t *testing.T) {
 	if !ok {
 		expect.Fail("Result is not content.IndexModel")
 	}
+	log.Println(model.Body)
```
Comment (todor-videv1) [inline pkg/message/message_handler_test.go:279]: **issue**: I think this might be unnecessary.

File: pkg/policy/agent_test.go
Comment (todor-videv1) [inline pkg/policy/agent_test.go:1]: **question**: There are several comments that are left over by the previous implementation, they do not seem to be relevant anymore. Stuff like "// query field is informative and is not being used during tests evaluation". Am I right?

File: .circleci/config.yml [hunk 4]
```diff
@@ -60,29 +65,43 @@ jobs:
...
+  test:
+    executor: default-ubuntu
+    working_directory: ~/content-rw-elasticsearch
+    steps:
+      - setup_remote_docker:
+          docker_layer_caching: true
+      - checkout
+      - run:
+          name: Run integration tests
+          command: export GITHUB_BRANCH=${CIRCLE_BRANCH} && docker-compose -f docker-compose-tests.yaml up -d --build && docker logs -f test-runner && export TESTS_STATUS_CODE=$(docker inspect test-runner --format="{{.State.ExitCode}}") &&  docker-compose -f docker-compose-tests.yaml down && exit $TESTS_STATUS_CODE
```
Comment (todor-videv1) [inline .circleci/config.yml:80]: **question**: Why do we set GITHUB_BRANCH?

**Review by todor-videv1 — CHANGES_REQUESTED** [review]
Setting this to "Request changes" so you can take a look at the comments, in any case it is an okay restructuring of the tests.

File: pkg/message/message_handler_test.go (hunk 2 above)
Comment (lokendersinghft) [reply pkg/message/message_handler_test.go:166]: yes..there should be a check and a default url. Also I am repeating this code in agent_test.go, so will extract it in other file. [Ref](https://github.com/Financial-Times/notifications-push/blob/master/access/evaluate_test_helper.go)

File: pkg/message/message_handler_test.go (hunk 3 above)
Comment (lokendersinghft) [reply pkg/message/message_handler_test.go:279]: It is unnecessary.. will be removed.

File: pkg/policy/agent_test.go
Comment (lokendersinghft) [reply pkg/policy/agent_test.go:1]: yes, you are right. These are unnecessary comments and  will be removed.

**Review by todor-videv1 — APPROVED** [review]

**Review by todor-videv1 — APPROVED** [review]

**Review by antoanFT — APPROVED** [review]

**Review by antoanFT — DISMISSED** [review]

**Review by antoanFT — APPROVED** [review]

**Review by todor-videv1 — APPROVED** [review]

**Pull request merged**